src/bin/observable.ts

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+import {join} from "node:path";
```

Comment:
```suggestion
import {join} from "node:path/posix";
```

I don’t think it matters here but stylistically we use posix by default and when we do need the os-specific behavior we use `import op from "node:path";`.